### PR TITLE
removed double activation of Source Verify

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-ide/issues/2877

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -398,7 +398,7 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
 
   await appManager.activatePlugin(['contentImport', 'theme', 'editor', 'fileManager', 'compilerMetadata', 'compilerArtefacts', 'network', 'web3Provider', 'offsetToLineColumnConverter'])
   await appManager.activatePlugin(['mainPanel', 'menuicons'])
-  await appManager.activatePlugin(['home', 'sidePanel', 'hiddenPanel', 'pluginManager', 'fileExplorers', 'settings', 'contextualListener', 'scriptRunner', 'terminal', 'fetchAndCompile'])
```
Comment (yann300) [inline src/app.js:401]: `sourceVerifier` and `fetchAndCompile` are 2 differents plugins

File: src/remixAppManager.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ import QueryParams from './lib/query-params'
 import { PermissionHandler } from './app/ui/persmission-handler'
 
 const requiredModules = [ // services + layout views + system views
-  'manager', 'compilerArtefacts', 'compilerMetadata', 'contextualListener', 'editor', 'offsetToLineColumnConverter', 'network', 'theme', 'fileManager', 'contentImport', 'web3Provider', 'scriptRunner', 'fetchAndCompile',
```
Comment (yann300) [inline src/remixAppManager.js:8]: what is the change here?

Comment (LianaHus) [reply src/remixAppManager.js:8]: just moved it to the new line

**Comment by yann300** [comment]
after discussion with @GrandSchtroumpf @LianaHus we decided to catch the error (see PR https://github.com/ethereum/remix-ide/pull/2924) and see in the future whether we need to have a more fine-grained error messages (probably using error codes).

**Pull request closed without merging**